cmd/launch.go

```diff
@@ -398,19 +430,17 @@ func ReadImageId(h *ec2helper.EC2Helper, simpleConfig *config.SimpleInfo, defaul
 Ask user input for the auto-termination timer.
 Return true if the function is executed successfully, false otherwise
 */
-func ReadAutoTerminationTimer(simpleConfig *config.SimpleInfo, defaultTimer int) bool {
+func ReadAutoTerminationTimer(h *ec2helper.EC2Helper, simpleConfig *config.SimpleInfo, defaultTimer int) bool {
 	// Ask for auto-termination timer
-	timerResponse := question.AskAutoTerminationTimerMinutes(defaultTimer)
-	if timerResponse != cli.ResponseNo {
-		timer, err := strconv.Atoi(timerResponse)
-		if cli.ShowError(err, "Asking auto-termination timer failed") {
-			return false
-		}
-		simpleConfig.AutoTerminationTimerMinutes = timer
-	} else {
-		simpleConfig.AutoTerminationTimerMinutes = 0
+	var timer int
+	timerResponse, err := question.AskAutoTerminationTimerMinutes(h, defaultTimer)
+	if err == nil {
+		timer, err = strconv.Atoi(timerResponse)
```

Comment:
The error check on line 437 is more about not going through with the integer conversion if there is an error with the question itself. Do you think it would be better to have this question, and others like it, have the question convert the string to an int and have it return the integer?

Reply:
Right, the error check on 437 is good. I'm referring to the fact that we have a new `err` object on line 438 that we never inspect.

But that's a good idea--if we know that the only valid answer for a particular kind of question is a number, we should return it that way at the lowest level possible. However, I'd say save that for a subsequent PR.

Reply:
If there is an error caused by lien 438 then it would be caught in line 440. I just left it there cause it wouldnt hurt, but its not really necessary.